cfgov/wellbeing/jinja2/wellbeing/results.html

```diff
@@ -461,89 +468,130 @@ <h3>Get on track for your financial future</h3>
+            <li>
+                <b>Un asesor de crédito de una organización sin fines de lucro</b>
+                puede ayudarle a hacer un plan de pago para sus tarjetas de
+                crédito y otras deudas. Estos son algunos
+                <a href="/es/obtener-respuestas/como-elijo-el-asesor-de-credito-adecuado-para-mi-es-1355/ ">
+                  consejos para escoger un asesor de crédito</a>.
             </li>
+            {% endif %}
             <li>
-                <b>A HUD-certified housing counselor</b>
-                can give advice if you’re having trouble paying your mortgage
-                or help you get ready for homeownership.
-                <a href="/find-a-housing-counselor/">Find one in your area.</a>
+                <b>{{ _('A HUD-certified housing counselor') }}</b>
+                {{ _('can give advice if you’re having trouble paying your mortgage') }}
+                {{ _('or help you get ready for homeownership.') }}
+                <a href="/find-a-housing-counselor/">{{ _('Find one in your area.') }}</a>
             </li>
             <li>
-                <b>A financial coach</b>
-                can assess your situation and work with you over time to resolve issues and get ahead.
+                <b>{{ _('A financial coach') }}</b>
+                {{ _('can assess your situation and work with you over time to resolve issues and get ahead.') }}
                 <a href="/about-us/blog/the-launch-of-the-cfpb-financial-coaching-initiative/">
-                    Find out about the CFPB coaching initiative.
+                {{ _('Find out about the CFPB coaching initiative.') }}
                 </a>
             </li>
             <li>
-                <b>Your local library</b>
-                can often connect you with organizations and people in your area.
+                <b>{{ _('Your local library') }}</b>
+                {{ _('can often connect you with organizations and people in your area.') }}
             </li>
         </ul>
     </div>
 
     <span id="compare"></span>
 
     <div class="block">
-        <h2>See how your score compares to group averages</h2>
+        <h2>{{ _('See how your score compares to group averages') }}</h2>
 
     {% if not user_score %}
         <div class="u-mb15">
         {{ notification.render(
             'warning',
             true,
-            'We never collect or store answers to the financial well-being questionnaire.',
-            'If you would like to see your score and answers on the page below,
-            please start with the <a href="../">questionnaire</a> and enter your responses.'
+            _('We never collect or store answers to the financial well-being questionnaire.'),
+            _('If you would like to see your score and answers on the page below, please start with the <a href="../">questionnaire</a> and enter your responses.')
         ) }}
-    </div>
+        </div>
     {% endif %}
```

Comment:
This one is not being translated, either.

Reply:
Addressed in newest commits.

Reply:
This one is still not being translated. I think you're going to have to do the same thing you did with the `current_language` conditional in the first of this kind of notification at the top of this page.

Reply:
Thanks for the sharp eyes! Fixed.